scripts/dev/build_release.py

```diff
@@ -342,7 +293,6 @@ def build_mac(
     utils.print_title("Building .app via pyinstaller")
     call_tox(f'pyinstaller-64bit{"-qt5" if qt5 else ""}', '-r', debug=debug)
     utils.print_title("Patching .app")
-    patch_mac_app(qt5=qt5)
     utils.print_title("Re-signing .app")
     sign_mac_app()
```

Comment:
I think we can get rid of the re-signing now that we don't patch anymore? Could you try to remove the re-signing part from `sign_mac_app` (so that it only verifies the signature), perhaps rename it to something more appropriate, and see what happens?

Reply:
![image](https://github.com/qutebrowser/qutebrowser/assets/7419144/f988f765-f867-4a51-9bcc-bd1f2d7c6b08)

Seems to work fine without, based on my extremely limited knowledge. It installed (drag over Applications shortcut after opening .dmg) over top of a previously nightly fine, and ran. Using codesign (-dv and -vv) on the application in a terminal yields the same result before and after. (The .dmg isn't signed, which seems to be a thing you can do these days.)